lib/browser.ts

```diff
@@ -962,17 +896,14 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    *
    * @param {number=} opt_timeout Number of milliseconds to wait for Angular to start.
    */
-  refresh(opt_timeout?: number) {
-    if (this.ignoreSynchronization) {
-      return this.driver.navigate().refresh();
+  async refresh(opt_timeout?: number) {
+    if (!await this.waitForAngularEnabled()) {
```

Comment:
`ignoreSynchronization` is problematic. It was previously noted as deprecated and I think we should remove this property.